src/middlewared/middlewared/plugins/docker_linux/update.py

```diff
@@ -0,0 +1,33 @@
+import middlewared.sqlalchemy as sa
+
+from middlewared.schema import Bool, Dict
+from middlewared.service import accepts, ConfigService
+
+
+class ContainerModel(sa.Model):
+    __tablename__ = 'services_container'
+
+    id = sa.Column(sa.Integer(), primary_key=True)
+    enable_image_updates = sa.Column(sa.Boolean(), default=True)
+
+
+class ContainerService(ConfigService):
+
+    class Config:
+        datastore = 'services.kubernetes'
+
+    @accepts(
+        Dict(
+            'container_update',
+            Bool('enable_image_updates'),
+            update=True,
+        )
+    )
+    async def do_update(self, data):
+        old_config = await self.config()
```

Comment:
Should we create a docstring here specifying the potential for rate throttling? Otherwise I see end-users just enabling this by default to check for latest updates for the images running their containers.

Reply:
We definitely should, thank you for pointing it out ;)